package.json

```diff
@@ -3,7 +3,9 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "build": "dotnet publish \"./src/view-object-builder/view-object-builder.csproj\" -c Release -o \"./.build\""
+	"build-win": "dotnet publish \"./src/view-object-builder/view-object-builder.csproj\" -c Release -o \"./.build-win\" --self-contained -r win-x64",
+	"build-linux": "dotnet publish \"./src/view-object-builder/view-object-builder.csproj\" -c Release -o \"./.build-linux\" --self-contained -r linux-x64",
+    "build": "npm run build-win && npm run build-linux"
```

Comment:
I don't think json cares but it looks like your indentation is off (or you've got a mix of tab vs space)

Reply:
Json doesn't care, you're right, but I did have the indentation wrong. I have no clue how I didn't notice, but now that I have I can't unsee. I'll fix it 😆